.gitignore

```diff
@@ -23,3 +23,6 @@ run
 
 # Files from Forge MDK
 forge*changelog.txt
+
+# image editing
+*.xcf
```

Comment:
I used GIMP to edit images, so it's kind of nice to just leave those files in the same directory..?